packages/grafana-data/src/utils/OptionsUIBuilders.ts

```diff
@@ -143,6 +143,21 @@ export class FieldConfigEditorBuilder<TOptions> extends OptionsUIRegistryBuilder
       settings: config.settings || {},
     });
   }
+
+  addGenericEditor<TSettings = any>(
```

Comment:
not strictly required for this PR (since the usage is commented out) If everything else is OK, i could also make this into a new PR (that uses this editor)